nuxeo-features/rest-api/nuxeo-rest-api-test/src/test/java/org/nuxeo/ecm/restapi/test/BatchUploadFixture.java

```diff
@@ -69,7 +68,8 @@
 @Features({ TransientStoreFeature.class, RestServerFeature.class })
 @Jetty(port = 18090)
 @RepositoryConfig(cleanup = Granularity.METHOD, init = RestServerInit.class)
-@Deploy("org.nuxeo.ecm.platform.restapi.test:multiblob-doctype.xml")
+@Deploy({"org.nuxeo.ecm.platform.restapi.test:multiblob-doctype.xml"
+        , "org.nuxeo.ecm.platform.restapi.test:test-conflict-batch-handler.xml"})
```

Comment:
Since [NXP-24366](https://jira.nuxeo.com/browse/NXP-24366) we now use one `@Deploy` annotation per contribution.